src/main/java/com/almanza/inventory/controller/impl/ShoeControllerImpl.java

```diff
@@ -0,0 +1,41 @@
+package com.almanza.inventory.controller.impl;
+
+import com.almanza.inventory.controller.ShoeController;
+import com.almanza.inventory.models.ShoeModel;
+import com.almanza.inventory.models.ShoeToAddModel;
+import com.almanza.inventory.models.ShoeToUpdateModel;
+import com.almanza.inventory.service.ShoeService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/shoes")
+@AllArgsConstructor
+public class ShoeControllerImpl implements ShoeController {
```

Comment:
please include JavaDoc for everything, you can use any plugin on intellij to help you